docs/user/tutorial-asgi.rst

```diff
@@ -967,6 +967,49 @@ adding ``--cov-fail-under=100`` (or any other percent threshold) to our
     tests in multiple environments would most probably involve running
     ``coverage`` directly, and combining results.
 
+Debugging ASGI Applications
+---------------------------
+(This section also applies to WSGI applications)
+
+While developing and testing ASGI applications, understanding how to configure and utilize logging can be helpful, especially when you encounter unexpected issues or behaviors.
+
+By default, Falcon does not set up logging for you, but Python's built-in `logging` module provides a flexible framework for emitting and capturing log messages. Here's how you can set up basic logging in your ASGI Falcon application:
```

Comment:
IIRC, we have intersphinx connected to the stdlib's docs, so you probably can even link to them:
```
:mod:`logging`
```

Reply:
Ah had not seen there was an intersphinx mapping. Doing this change.